server/src/webserver.rs

```diff
@@ -844,3 +987,33 @@ pub async fn main() -> std::io::Result<()> {
     .run()
     .await
 }
+
+// <h2>&nbsp;</h2>
+// <h2>Tests</h2>
+// <p>As mentioned in the lexer.rs tests, Rust <a
+//         href="https://doc.rust-lang.org/book/ch11-03-test-organization.html">almost
+//         mandates</a> putting tests in the same file as the source. Here's
+//     some <a
+//         href="http://xion.io/post/code/rust-unit-test-placement.html">good
+//         information</a> on how to put tests in another file, for future
+//     refactoring reference.</p>
+// <p>&nbsp;</p>
+#[cfg(test)]
+
+
+// <p>Provide a way to create a CodeDocBlock</p>
+
+
+    
+// <h3>Save Endpoint Testing</h3>
+
+#[test]
```

Comment:
Put all tests in side a `mod test { ... }` like the lexer's tests.

Reply:
Added!